src/app/themes.tsx

```diff
@@ -11,6 +11,9 @@ import {
 } from "styled-components";
 import MaterialAndGlobalStyle from "@/app/global_styles";
 import useLocalStorage from "@/components/Hooks/useLocalStorage";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
```

Comment:
can this stuff go in `layout.tsx` instead of `themes.tsx`? it's not really a theming thing to setup providers and context :)